app/serializers/provider_serializer.rb

```diff
@@ -113,4 +105,17 @@ def select_value_for_provider(provider_code, values)
     index = provider_code.to_i(36) % values.count
     values[index]
   end
+
+  def generate_provider_contacts
+    object.contacts.map { |c| c.attributes.slice('type', 'name', 'email', 'telephone') }
+  end
+
+  def add_application_alert_recipient
```

Comment:
maybe not `add_something` as its not really adding....

Reply:
And done. Thanks for the thorough feedback @defong.